src/controllers/users.controller.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const usersService = require('../services/users.service');
+
+const getAll = async(req, res) => {
+  const users = await usersService.getAll();
+
+  res.send(
+    users.map(usersService.normalize)
+  );
+};
+
+const getOne = async(req, res) => {
+  const { userId } = req.params;
+  const foundUser = await usersService.getById(userId);
```

Comment:
Let's check the required data before using business logic
```suggestion
  const { userId } = req.params;
  
  if (!userdId) {
    some error
  } 
  
  const foundUser = await usersService.getById(userId);
```